app/models/task.py

```diff
@@ -1,6 +1,30 @@
 from flask import current_app
 from app import db
 
-
 class Task(db.Model):
-    task_id = db.Column(db.Integer, primary_key=True)
+    task_id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    title = db.Column(db.String)
+    description = db.Column(db.String)
+    completed_at = db.Column(db.DateTime, nullable = True)
+    goal_id = db.Column(db.Integer, db.ForeignKey("goal.goal_id"), nullable=True)
+
+    def is_complete(self):
+        return self.completed_at != None
+
+    
+    def make_json(self):
```

Comment:
Good use of a helper method. Consider DRYing up your code by building the dictionary 

```
json = 
{"id": self.task_id,                    
"title": self.title,
 "description": self.description,
"is_complete": self.is_complete()}
```
and conditionally adding the `goal_id` like this:

```
if self.goal_id:
    json["goal_id] = self.goal_id
```

and then returning `{"task": json}`